config/sample.config.yaml

```diff
@@ -29,6 +29,9 @@ app:
   # secret string "val://user:password"
   # optional
   resources_config_path_secret: env://TEST_RESOURCE_CONFIG_SECRET
+  # org_disable_on_creation if set to true will set the org status to disabled on creation. This can be used to
+  # prevent users from accessing the org until they contact the admin and get it enabled. Default is false
+  org_disable_on_creation: false
```

Comment:
let's rephrase this as `disable_orgs_on_create` to keep it consistent with the existing config?